### PR TITLE
Continuserv: Reduce number of watches

### PR DESCRIPTION
**Pull request by richvdh**
rather than trying to watch half the filesystem, be a bit more conservative in
what we put watches on. This means that we don't end up dying with "No space on
device" (thanks for that, Linux.)

### CODE REVIEW

**Comment by richvdh** [comment]
Assigning to @NegativeMjark as he is probably most likely to pick up on the Go style points at :)

File: scripts/continuserv/main.go [hunk 1]
```diff
@@ -95,20 +115,12 @@ func filter(e fsnotify.Event) bool {
...
 		return false
 	}
+
 	// Avoid some temp files that vim writes
 	if strings.HasSuffix(e.Name, "~") || strings.HasSuffix(e.Name, ".swp") || strings.HasPrefix(e.Name, ".") {
 		return false
 	}
 
-	// Ignore the .git directory - It's very noisy
-	if strings.Contains(e.Name, "/.git/") {
```
Comment (Kegsay) [inline scripts/continuserv/main.go]: Why are we deleting this?

**Comment by Kegsay** [comment]
LGTM modulo comment with my limited Go knowledge

File: scripts/continuserv/main.go (hunk 1 above)
Comment (richvdh) [inline scripts/continuserv/main.go]: because the changes at line 53ish mean that we no longer end up adding watchers which include the .git directory.

**Comment by NegativeMjark** [comment]
LGTM

File: scripts/continuserv/main.go (hunk 1 above)
Comment (Kegsay) [inline scripts/continuserv/main.go]: Derp, LGTM!

**Pull request merged**